lms/static/sass/shared/_footer-edx.scss

```diff
@@ -392,24 +595,96 @@ footer#footer-edx-v3 {
       @include margin-left($baseline*0.25);
     }
 
-    .social {
-      display: inline;
-      margin-top: 0;
+    .footer-language-selector {
+      display: inline-block;
+      float: none;
     }
+  }
 
+  @include media-breakpoint-up(md) {
+    .ml-auto {
+      margin-left: auto !important;
```

Comment:
It would be helpful for future development to include a comment explaining why ```!important``` is being used here.